src/Components/Table.js

```diff
@@ -1,16 +1,18 @@
 import React, { useState } from "react";
 import PopUp from "./PopUp";
-import { useTable, useGlobalFilter, useFilters, useSortBy } from "react-table"; //React table documentation https://react-table.tanstack.com/
+import { useTable, useGlobalFilter, useFilters, useSortBy } from "react-table"; 
 import { GlobalFilter, DefaultColumnFilter, SortOnClick } from "./Filters";
 
-// functions marked # are snippets from https://react-table.tanstack.com/docs/examples/filtering without modifications
-// functions marked ## are snippets with our own modifications
-// functions marked ### are OC 
+// Created a table based on input. Used in our pages.
+// We've split it into Table and TableRow to make it easier to manage. Could still be more optimized however.
+// Most styling still in here, need to create a functioning stylesheet.
 
-export default function Table({ columns, data }) {   // ##
-  // Table component logic and UI come here
+// Code based on react-table [https://react-table.tanstack.com/] 
+// & table-filters [https://react-table.tanstack.com/docs/examples/filtering]
 
-  const filterTypes = React.useMemo( //#
+export default function Table({ columns, data, color }) {   
+
+  const filterTypes = React.useMemo( 
```

Comment:
You are not entirely consistent in how you do your imports and therefor your code is a little messy.

Either use destructuring/named imports:
`import { useState, useMemo } from "react";
//...
const filterTypes = useMemo(//whatever you need to put here)
//...`

or default imports:
`import React from "react";
//...
const filterTypes = React.useMemo(//whatever you need to put here)
//...`
You seem to mostly use named imports... :)